airbyte-integrations/connector-templates/source-python/sample_files/invalid_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "fix-me-too": "value"
```

Comment:
everything that a user should fill in should have a `TODO` --  in the instructions we're going to tell them to grep the project for "TODO" and apply the instructions